Source/Shaders/SkyAtmosphereVS.glsl

```diff
@@ -126,5 +127,6 @@ void main(void)
     v_mieColor = frontColor * KmESun;
     v_rayleighColor = frontColor * (InvWavelength * KrESun);
     v_toCamera = czm_viewerPositionWC - positionV3;
+    v_radii = u_cameraAndRadiiAndDynamicAtmosphereColor.zy;
```

Comment:
Why pass this in as a varying, when the FS also has access to the same uniform?

/cc @abwood